### PR TITLE
Extends the title of the agenda-items

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- The title of motions are shown.
- If motions have a comment, an icon will be shown.
- An error caused by the subtitle is catched.

### CODE REVIEW

File: client/src/app/core/repositories/motions/motion-repository.service.ts [hunk 1]
```diff
@@ -296,8 +302,12 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
     /**
      * @override The base function and returns the submitters as optional subtitle.
      */
-    public getAgendaSubtitle = (model: ViewMotion) => {
-        return model.submittersAsUsers.join(', ');
+    public getAgendaSubtitle = (motion: ViewMotion) => {
+        if (motion.submittersAsUsers.length > 0) {
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/motions/motion-repository.service.ts]: ` if (motion.submittersAsUsers.length)`

File: client/src/app/core/repositories/motions/motion-repository.service.ts [hunk 2]
```diff
@@ -287,7 +287,13 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
...
         // Append the verbose name only, if not the special format 'Motion <identifier>' is used.
         if (titleInformation.identifier) {
-            return numberPrefix + this.translate.instant('Motion') + ' ' + titleInformation.identifier;
+            return (
+                numberPrefix +
+                this.translate.instant('Motion') +
+                ' ' +
+                titleInformation.identifier +
+                ` · ${titleInformation.title}`
+            );
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/motions/motion-repository.service.ts]:
```
return `${numberPrefix} ${this.translate.instant('Motion')} ${titleInformation.identifier} · ${titleInformation.title}`
```

File: client/src/app/core/repositories/motions/motion-repository.service.ts [hunk 3]
```diff
@@ -287,7 +287,13 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
...
         } else {
             return numberPrefix + titleInformation.title + ' (' + this.getVerboseName() + ')';
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/motions/motion-repository.service.ts]: while we are at it, please also convert this to template strings using ticks

File: client/src/app/core/repositories/base-is-agenda-item-and-list-of-speakers-content-object-repository.ts [hunk 4]
```diff
@@ -59,14 +59,7 @@ export abstract class BaseIsAgendaItemAndListOfSpeakersContentObjectRepository<
...
 
-    /**
-     * Overwrites the base function.
-     *
-     * @param titleInformation The information about the model.
-     *
-     * @returns {string | null} An optional subtitle. `Null`, if it returns no subtitle, otherwise `string`.
-     */
-    public getAgendaSubtitle(titleInformation: T): string | null {
+    public getAgendaSubtitle(viewModel: V): string | null {
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/base-is-agenda-item-and-list-of-speakers-content-object-repository.ts]: please don't just delete comments

File: client/src/app/core/repositories/motions/motion-repository.service.ts [hunk 5]
```diff
@@ -287,17 +287,23 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
...
     };
 
     /**
      * @override The base function and returns the submitters as optional subtitle.
      */
-    public getAgendaSubtitle = (model: ViewMotion) => {
-        return model.submittersAsUsers.join(', ');
+    public getAgendaSubtitle = (motion: ViewMotion) => {
+        if (motion.submittersAsUsers && motion.submittersAsUsers.length) {
+            return this.translate.instant('by') + ` ${motion.submittersAsUsers.join(', ')}`;
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/motions/motion-repository.service.ts]:
```
return `${this.translate.instant('by')} ${motion.submittersAsUsers.join(', ')}`
```

**Comment by tsiegleauq** [comment]
Code looks okay now.
@emanuelschuetze since the return values of the strings changed, this should be tested again. Can be merged after

**Comment by emanuelschuetze** [comment]
Successfully retested.

**Pull request merged**